pages/api/user/post.ts

```diff
@@ -0,0 +1,14 @@
+import { User } from "database/models/user";
+import { NextApiRequest, NextApiResponse } from "next";
+import getAll from "./getAll";
+
+const post = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    await User.create(req.body)
```

Comment:
should we do a check here to make sure we aren't creating a "super admin" if one already exists (since there should only be one i think)